src/OrbitBase/AnyInvocableTest.cpp

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <gtest/gtest.h>
+
+#include "OrbitBase/AnyInvocable.h"
+
+namespace orbit_base {
+
+static int returns_magic_number() { return 42; }
```

Comment:
super nitpick: this is not a magic number either change it to the random number: https://xkcd.com/221, or rename this to get_answer_to_life_universe_and_everything()

Reply:
Oh yes. You're so right about this. :smile: Apologies. Fixed!